scripts/combine/setupCombineWMass.py

```diff
@@ -5,16 +5,22 @@
 import argparse
 import os
 import pathlib
+import logging
+
+logging.basicConfig(level=logging.INFO)
 
 scriptdir = f"{pathlib.Path(__file__).parent}"
 
 parser = argparse.ArgumentParser()
 parser.add_argument("-o", "--outfolder", type=str, default="/scratch/kelong/CombineStudies")
 parser.add_argument("-i", "--inputFile", type=str, required=True)
-parser.add_argument("--qcdScale", choices=["byHelicityPt", "byPt", "integrated"], default="byHelicityPt", 
+parser.add_argument("--qcdScale", choices=["byHelicityPt", "byPt", "integrated", "ByCharge"], default="byHelicityPt", 
```

Comment:
What does the "ByCharge" option actually do here (aren't they always split by charge?)

Reply:
Yeah, in principle this was going to be an option but it isn't for now. Will remove.